apps/admin/frontend/src/screens/logs_screen.tsx

```diff
@@ -8,6 +9,13 @@ export function LogsScreen(): JSX.Element {
   const { electionDefinition, usbDriveStatus, auth, logger, machineConfig } =
     useContext(AppContext);
 
+  const exportLogsToUsbMutationAsync =
+    exportLogsToUsb.useMutation().mutateAsync;
```

Comment:
Question for you @jonahkagan: Are we still recommending usage of `.mutate` instead of `.mutateAsync` (unless wrapped in a try-catch) for error handling purposes, as documented on [this slide](https://docs.google.com/presentation/d/1qcohhkeavCVjTZAAhGbbB7sbnvDJLlNov3OTLijfcvs/edit#slide=id.g1f8414d696e_0_361)?

Reply:
Yes, that's still the recommendation. This seems like a case where the consuming code needs to await the mutation, so using `mutateAsync` makes sense. But it does still need to be wrapped in a try/catch to avoid unhandled promise crashes.

Also, I would say you don't need the `useCallback` here - I don't think there will be any performance issue without it.